### ISSUE TITLE
 Unknown block type

### ISSUE THREAD

**Issue by kxs-mdoyon** [inkdrop-org/inkdrop-visualizer#54]
Hello, It seem external module not supported still?

Computing terraform graph...
Diagram renderer running on localhost:3000
LOG: Detailed view is off.
LOG: Unchanged resources are shown.
LOG: Full opacity for unchanged resources is off.
ERR: DotSyntaxError: Expected "#", "/*", "//", "digraph", "graph", or "strict" but end of input found.
Creating SVG of the diagram...
LOG: Detailed view is off.
LOG: Unchanged resources are shown.
LOG: Full opacity for unchanged resources is off.
LOG: Aggregating multiple states graphs...
LOG: Terraform plan detected.
LOG: Adding main resources...
WARN: Unknown block type auth0_tenant.tenant
WARN: Unknown block type auth0_branding.branding
WARN: Unknown block type auth0_branding_theme.branding_theme
WARN: Unknown block type auth0_prompt.prompt
WARN: Unknown block type auth0_prompt_custom_text.prompt_custom_text
WARN: Unknown block type auth0_custom_domain.custom_domain
WARN: Unknown block type auth0_custom_domain_verification.custom_domain_verification
WARN: Unknown block type vault_generic_secret.custom_domain_verification_records
WARN: Unknown block type auth0_email_provider.email_provider
WARN: Unknown block type auth0_log_stream.eventgrid
WARN: Unknown block type auth0_email_template.email_template
WARN: Unknown block type auth0_log_stream.datadog
LOG: Aggregating secondary resources...
LOG: Aggregating secondary resources... Done.
WARN: Unknown block type auth0_attack_protection.attack_protection
WARN: Unknown block type auth0_guardian.guardian
LOG: Adding main resources... Done.
LOG: Removing inactive resources... Done.
LOG: Computing connections...
LOG: Computing connections... Done.
LOG: Removing inactive resources...
Writing 'inkdrop-ci-data_24-06-04_12-46-39.json'...
Empty diagram detected. No SVG will be created.

**Comment by dant2021** [comment]
Yes, we currently still only support AWS, GCP and Azure providers

**Comment by kxs-mdoyon** [comment]
Thanks for the answer. Any plan to support other providers? :)

**Comment by danielskowronski** [comment]
Given fact there's Terraform provider for nearly everything, including pizza, support for any type of resource (even as opt-in feature) is much needed. If tthat's not planned, please at least add warning into README.md 

This is really about `any` type of resource, not just adding support for subsequent cloud providers (although those would be nice as well).
